source/standards/understanding-risks.html.md.erb

```diff
@@ -19,7 +21,16 @@ The Service Manual has some recommendations which can reduce risk to your servic
 
 [Modelling threats][] can help you gain a clearer understanding of threats against your service. GDS uses [Attack Tree][] development workshops to model threats. Any workshops you run should cover all potential [attack vectors][].
```

Comment:
I think we should perhaps cross-link here to the GDS Way page on threat modelling instead of the OWASP page